delivery_admin/views/react/orders.jsx

```diff
@@ -2,27 +2,78 @@ import React from 'react'
 import $ from 'jquery'
 const date = require('../../../tools/date.js')()
 const globals = require('../../../tools/globals.js')
+const _ = require('underscore')
 
 class Orders extends React.Component {
 
   constructor (props) {
     super(props)
+    let current_date = date.this_delivery().format(globals.order_date_format).slice(0,-6)
     this.state = {
       orders: [],
+      shown_date: current_date
     }
+    this.handle_date_change = this.handle_date_change.bind(this)
   }
 
   componentWillMount () {
-    $.get("/delivery_orders?date="+date.this_delivery().format(globals.order_date_format), (data) => {
-      this.setState({orders:data})
+    $.get("/delivery_orders?date=" + this.state.shown_date, (data) => {
+      this.setState({orders: data})
+    })
+  }
+
+  count_orders (orders) {
+    let count = {}
+    _.each(orders, (order) => {
+      let meal = order.meal
+      count[meal] = count[meal]
+        ? count[meal] + 1
+        : 1
+    })
+    return count
+  }
+
+  handle_date_change(){
+    let chosen_date = document.getElementById("chosen_date").value + " 12:00"
```

Comment:
Adding 12 to the date will work for now. We have to discuss a solution that is more generic to make sure we can set delivery times specifically for one station. My code does not support this yet either, but it should not be to difficult to change it.